lib/python/bind_slice_methods.h

```diff
@@ -184,15 +184,33 @@ template <class T> struct slicer {
   }
 };
 
+namespace {
+void expect_implicit_dimension(const Sizes &dims) {
+  using std::to_string;
+  if (dims.size() == 0)
+    throw except::DimensionError("Slicing scalar object is not possible.");
```

Comment:
I think it has to be either 'a scalar object' or 'scalar objects' (non-native though...)

Reply:
```suggestion
    throw except::DimensionError("Slicing a scalar object is not possible.");
```